src/Compressors/Gaussian.jl

```diff
@@ -0,0 +1,98 @@
+struct Gaussian <: Compressor
+    n_rows::Int64
+    n_cols::Int64
```

Comment:
Should have an internal constructor that checks the validity of the input arguments.